docs/source/supported/rivanna.rst

```diff
@@ -0,0 +1,112 @@
+====================
+Rivanna (UVA)
+====================
+
+Platform user guide
+===================
+
+https://www.rc.virginia.edu/userinfo/user-guide/
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``uva.rivanna*`` - ``MPIRUN``
+
+* Configuration per node (per platform ID)
+
+  * ``uva.standard`` (1 node)
+
+    * 40 CPU cores (1 thread per core)
+    * 375GB GiB of memory
+
+  * ``uva.parallel`` (25 nodes)
+
+    * 40 CPU cores (1 thread per core)
+    * 375GB GiB of memory
+
+  * ``uva.largemem`` (1 node)
+
+    * 10 CPU cores (1 thread per core)
+    * 975 GiB of memory
+
+  * ``uva.dev`` (2 nodes)
+
+    * 4 CPU cores (1 thread per core)
+    * 36 GiB of memory
+
+  * ``uva.gpu`` (4 nodes)
+
+    * 10 CPU cores (1 thread per core)
+    * 16 GPUs NVIDIA(RTX2080Ti, RTX3090, K80, P100, V100, and A100)

```

Comment:
is this correct? more types of GPUs than the amount of nodes?

Reply:
This looks over-simplified. The nodes are heterogeneous.

Ref https://www.rc.virginia.edu/userinfo/rivanna/slurm/#gpu-computations though much of the rivanna documentation online is out of date and inconsistent.

According to `qlimits`
* up to 32 gpus per job
* up to 4 nodes per job

I found the following table at https://www.rc.virginia.edu/userinfo/rivanna/overview/#system-details, which may or may not be current:


Cores/Node | Memory/Node | Specialty Hardware | GPU memory/Device | GPU devices/Node | # of Nodes
-- | -- | -- | -- | -- | --
128 | 1000GB | GPU: A100 | 40GB | 8 | 2
128 | 2000GB | GPU: A100 | 80GB | 8 | 11
28 | 255GB | GPU: K80 | 11GB | 8 | 8
28 | 255GB | GPU: P100 | 12GB | 4 | 4
40 | 383GB | GPU: RTX2080Ti | 11GB | 10 | 2
64 | 128GB | GPU: RTX3090 | 24GB | 4 | 5
28 | 188GB | GPU: V100 | 16GB | 4 | 1
40 | 384GB | GPU: V100 | 32GB | 4 | 12
36 | 384GB | GPU: V100 | 32GB | 4 | 2

Reply:
8 GPUs on a 28 core node is a fun one to allocate, eh?